auth-layer-proxy/scripts/start-envoy.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# Set default values for environment variables if they are not already set
+export SERVICE_TYPE=${SERVICE_TYPE:-LOGICAL_DNS}
+export SERVICE_ADDRESS=${SERVICE_ADDRESS:-host.docker.internal}
+export SERVICE_PORT=${SERVICE_PORT:-8020}
+export ENVOY_ADMIN_PORT=${ENVOY_ADMIN_PORT:-15000}
+export PROXY_PORT=${PROXY_PORT:-10000}
+export CLIENT_ID=${CLIENT_ID:-htg-auth-layer}
+export TOKEN_INTROSPECTION_URL=${TOKEN_INTROSPECTION_URL:-http://host.docker.internal:8080/realms/HederaTheGraph/protocol/openid-connect/token/introspect}
+
+# print the environment variables
+echo "----------------------------------------"
+echo "---  Environment variables:          ---"
+echo "----------------------------------------"
+echo "SERVICE_TYPE: $SERVICE_TYPE"
+echo "SERVICE_ADDRESS: $SERVICE_ADDRESS"
+echo "SERVICE_PORT: $SERVICE_PORT"
+echo "ENVOY_ADMIN_PORT: $ENVOY_ADMIN_PORT"
+echo "PROXY_PORT: $PROXY_PORT"
+echo "CLIENT_ID: $CLIENT_ID"
+echo "CLIENT_SECRET: $CLIENT_SECRET"
```

Comment:
Q: should we be echoing this?

Reply:
yes, we should not log the secret, maybe we can obfuscate it and only show the last 4 bytes that could be useful for troubleshooting and make sure that the `auth-layer-proxy` is using the correct configuration values.

Reply:
decided to go for the first 4 chars:

it looks something like this:

```bash
CLIENT_SECRET: 0cyY******
```

✅  Done! 👍